css/css-backgrounds/border-radius-clipping-002.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Backgrounds and Borders Test: 'border-radius' and corner-clipping of content elements</title>
+
+  <link rel="author" title="zhouli" href="mailto:liz@oupeng.com">
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="http://www.w3.org/TR/css3-background/#corner-clipping">
+  <link rel="match" href="../reference/ref-nothing-below.xht">
+
+  <meta content="" name="flag">
+  <meta name="assert" content="When set with 'overflow: hidden', an outer container must clip background of its child element at its corners. In this test, the outer container, which has immense size, has a 'border-radius' declaration which implies that the clipping must occur at its curve.">
```

Comment:
"background" here should be "text" since we are clipping text, not the background.

Reply:
Yes. Agreed. I should reformulate this better.